Add Makefiles to make building easier.

waltje opened the pull request.

boeckmann commented:
In fact I had the ROMs in but threw them out because of some legal issues here in Germany. Do not want to get a letter from a lawyer because of this :-)

I incorporated some of your changes manually. Your makefiles are in my repository under the name Makefile.vasm. Mine is called Makefile.asm6502.

I incorporated an MD5 check into the build process. Perhaps you want to adapt that.

boeckmann commented on kernal/editor1.a65:
```diff
@@ -69,7 +69,7 @@ CLSR	LDA HIBASE      ;FILL HI BYTE PTR TABLE
 	LDA #0
 	TAX
-LPS1	STY LDTB1,X
+LPS1	STY [b]LDTB1,X
```
Is this by intention? LDTB1 evaluates to $D9, so that should not be necessary (does not hurt either).

boeckmann commented on .gitignore:
Added this by hand.

boeckmann commented on basic/Makefile:
Added as Makefile.vasm

boeckmann commented on kernal/Makefile:
Added as Makefile.vasm

boeckmann commented on rom/Makefile:
added as Makefile.vasm

boeckmann commented on rom/good/basic.901226-01.bin:
Sadly legal issues, would like to included but am afraid of.

boeckmann commented on rom/good/kernal.901227-03.bin:
Sadly legal issues, would like to included but am afraid of.

boeckmann commented on supp/Makefile:
```diff
@@ -0,0 +1,10 @@
+
+CC	:= gcc
```
Is this strictly necessary? Normally CC should be pre-defined?

waltje commented:
oh  thats a leftover, sorry.--Sent via my Samsung Galaxy Note® 4 4G LTE

-------- Original message --------
From: Bernd Böckmann ***@***.***> 
Date: 4/23/23  8:24 AM  (GMT-05:00) 
To: boeckmann/c64krnl ***@***.***> 
Cc: "Fred N. van Kempen" ***@***.***>, Author ***@***.***> 
Subject: Re: [boeckmann/c64krnl] Add Makefiles to make building easier. (PR
&nbsp; #1) 


@boeckmann commented on this pull request.



In kernal/editor1.a65:
> @@ -69,7 +69,7 @@ CLSR	LDA HIBASE      ;FILL HI BYTE PTR TABLE
 	TAY
 	LDA #0
 	TAX
-LPS1	STY LDTB1,X
+LPS1	STY [b]LDTB1,X

Is this by intention? LDTB1 evaluates to $D9, so that should not be necessary (does not hurt either).



On .gitignore:
Added this by hand.



On basic/Makefile:
Added as Makefile.vasm



On kernal/Makefile:
Added as Makefile.vasm



On rom/Makefile:
added as Makefile.vasm



On rom/good/basic.901226-01.bin:
Sadly legal issues, would like to included but am afraid of.



On rom/good/kernal.901227-03.bin:
Sadly legal issues, would like to included but am afraid of.



In supp/Makefile:
> @@ -0,0 +1,10 @@
+
+CC	:= gcc

Is this strictly necessary? Normally CC should be pre-defined?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/boeckmann/c64krnl/pull/1#pullrequestreview-1396899942",
"url": "https://github.com/boeckmann/c64krnl/pull/1#pullrequestreview-1396899942",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
